### ISSUE TITLE
fb.schedule module not working for 2020-21?

### ISSUE THREAD

**Issue by tonyelhabr** [roclark/sportsreference#490]
**Describe the bug**
Schedule for fb module for 2020-21 season does not return data.

**To Reproduce**
```python
from sportsreference.fb.schedule import Schedule

tottenham = Schedule('Tottenham Hotspur')
for game in tottenham.schedule:
    print(game.datetime)  # Prints the datetime for each game
```

**Expected behavior**
Return datetimes of games

**Screenshots**
Output that I'm currently getting
```
The requested page returned a valid response, but no data could be found. Has the season begun, and is the data available on www.sports-reference.com?
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Sportsreference Version 0.5.2

**Additional context**
The team module seems to work fine.

Not sure if Roster is working either.

```python
from sportsreference.fb.roster import Roster

rost = Roster('Tottenham Hotspur')
print(rost.__dict__)
```

```
{'_players': [], '_squad_id': '361ca564'}
```

Let me know if I can provide more info. Thanks!

**Comment by vatsal-M** [comment]
Our team @ISD14-LNMIIT wants to work on this issue. Please assign this issue to us!
@tonyelhabr

**Comment by tonyelhabr** [comment]
@vatsal-M awesome! am i supposed to assign you the issue? it's not clear to me how to do that

**Comment by vatsal-M** [comment]
@tonyelhabr  I think you should be able to assign this issue to @ISD14-LNMIIT as you have opened this issue. @roclark can you look into this situation. 
We are a team of college undergrads and we have to solve an issue on Github as our project. We are following the team software process and as such it will take about a month for us to resolve the issue but we will ensure high-quality work.

**Comment by roclark** [comment]
Hey all, thanks for bringing this to my attention and for offering to support! This has actually been fixed as of #480 as the tables on sports-reference.com were updated and the old method of pulling information was broken. With the latest version of the upstream code, both the `Schedule` and `Roster` (plus `Team`) classes should work as expected with valid output. These updates will be included with the 0.6.0 release which I hope to put out relatively soon. If you desire to use the updated code now, you can pull/build the source according to the README. I will be closing this given the mentioned update, but if there are further issues, feel free to re-open or create a new issue and I will look into it.

**Comment by ericmk52** [comment]
Hi, 

I am not sure if you have used the sportsipy package recently, but when I try to run the current schedules for teams, I am receiving the same error. Do you know if there is a fix?

Thanks,
Eric
